murdock/job.py

```diff
@@ -79,6 +79,14 @@ def remove_dir(work_dir):
         except FileNotFoundError:
             LOGGER.debug(f"Directory '{work_dir}' doesn't exist, cannot remove")
 
+    @property
+    def uid(self) -> str:
+        return self._uuid.hex
```

Comment:
```suggestion
        return self.uuid.hex
```

And line 88 should be covered by the tests.

Reply:
Fixed and added a test case